### PR TITLE
update version and fix a deployment sample

### PR DESCRIPTION
**Pull request by dtrawins**

### CODE REVIEW

**Review by rasapala — APPROVED** [review]
LGTM

File: config/samples/intel_v1alpha1_ovms.yaml [hunk 1]
```diff
@@ -17,7 +17,7 @@ spec:
   models_settings:
     single_model_mode: true
     config_configmap_name: ""
-    model_config: ""   
+    model_config_path: ""
```
Comment (mzegla) [inline config/samples/intel_v1alpha1_ovms.yaml:20]: I think it's just `config_path`:
https://github.com/openvinotoolkit/operator/blob/main/bundle/manifests/openvino-operator.clusterserviceversion.yaml#L109-L115

File: bundle_k8s/manifests/openvino-operator.clusterserviceversion.yaml [hunk 2]
```diff
@@ -510,5 +510,5 @@ spec:
   maturity: alpha
   provider:
     name: Intel
-  version: 1.0.0
-  replaces: ovms-operator.v0.2.0
+  version: 1.1.0
+  replaces: openvino-operator.v1.0.0
```
Comment (mzegla) [inline bundle_k8s/manifests/openvino-operator.clusterserviceversion.yaml:514]: v1.1.0?

File: Makefile [hunk 3]
```diff
@@ -129,7 +129,7 @@ ifeq ($(TARGET_PLATFORM), openshift)
     fi
 	docker tag $(CATALOG_REPOSITORY):$(IMAGE_TAG) $(CATALOG_REPOSITORY):$(BRANCH)-latest 
 else
-	sudo opm index add --bundles quay.io/operatorhubio/ovms-operator:v0.1.0,quay.io/operatorhubio/ovms-operator:v0.2.0,$(BUNDLE_REPOSITORY)-k8s:$(IMAGE_TAG) -c docker --tag $(CATALOG_REPOSITORY)-k8s:$(IMAGE_TAG)
+	sudo opm index add --bundles quay.io/operatorhubio/ovms-operator:v0.1.0,quay.io/operatorhubio/ovms-operator:v0.2.0,quay.io/operatorhubio/ovms-operator:v1.0.0,$(BUNDLE_REPOSITORY)-k8s:$(IMAGE_TAG) -c docker --tag $(CATALOG_REPOSITORY)-k8s:$(IMAGE_TAG)
```
Comment (mzegla) [inline Makefile:132]: v1.1.0?

File: config/samples/intel_v1alpha1_ovms.yaml (hunk 1 above)
Comment (mzegla) [reply config/samples/intel_v1alpha1_ovms.yaml:20]: same here:
https://github.com/openvinotoolkit/operator/blob/main/helm-charts/ovms/values.yaml#L28

**Review by atobiszei — APPROVED** [review]

**Review by mzegla — APPROVED** [review]

**Pull request merged**